MekHQ/src/mekhq/gui/StratconPanel.java

```diff
@@ -356,9 +397,17 @@ private boolean drawHexes(Graphics2D g2D, DrawHexType drawHexType) {
                     }
                 }
 
+                // here we draw the coordinate labels
                 if (drawHexType == DrawHexType.Hex) {
                     g2D.setColor(Color.GREEN);
```

Comment:
If you want, it should be possible to replace 402-410 with 
`new StringDrawer(currentCoords.toBTString())
.at(graphHex.xpoints[0] + xRadius / 2, graphHex.ypoints[0] + ((int) (g2D.getFontMetrics().getHeight() / 1.25)))
.centerX()
.color(Color.GREEN)
.font(g2D.getFont().deriveFont(Font.BOLD, g2D.getFont().getSize()))
.draw(g2D);`

I had to guess for the x center, but the coords probably look better when x-centered. Also I struggled a bit with the font, as it is not set anywhere. I dont know what the font size is then, I'd guess 12. It could simplify things if the font size were set, e.g. at yRadius/3

```
float fontSize = yRadius/3;
new StringDrawer(currentCoords.toBTString())
.at(graphHex.xpoints[0] + xRadius / 2, graphHex.ypoints[0] + (int)(fontSize*1.2))
.center()
.color(Color.GREEN)
.font(g2D.getFont().deriveFont(Font.BOLD))
.fontSize(fontSize)
.draw(g2D);
```
The font and fontSize could be set outside of the loop, making the call simpler.